Automattic/wp-calypso
# [Login] Connect the login form to WPCOM

Requires server patch D5159

This PR updates the `wp-login` component to issue an authentication request to the wpcom server. Once the user's credentials are checked to be valid, we issue a synchronous post request to `wp-login.php` with those same credentials.

### Testing Instructions
- Checkout this branch and apply D5159 on your sandbox.
- Change your hosts file to point `wordpress.com` to your sandbox.
- Launch calypso locally, check that you are logged out and navigate to http://calypso.localhost:3000/login
- Enter your username/email and password and click on `Log In`
- Assert that you are redirected to http://calypso.localhost:3000/ and that you are now logged in.
- Check with invalid credentials

### Reviews
